packages/shared/src/components/squads/layout/SquadDirectoryLayout.tsx

```diff
@@ -69,46 +69,45 @@ export const SquadDirectoryLayout = (
 
   const isDiscover = pathname === squadCategoriesPaths.discover;
 
+  if (!loadedSettings) {
```

Comment:
Came across this. Nothing changed here, just made an early return to reduce the indents.